.github/workflows/deploy-template_demo.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy Template Demo
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: The environment to deploy to.
+        required: true
+        default: production
+jobs:
+  Deploy:
```

Comment:
Will this workflow be different than our current deploy workflow which updates the manifest image tag and pushes to ECR?

Reply:
No, this is just an example to demo the flow of everything. We will be able to use whichever workflow we choose.